src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,15 +1,18 @@
 package mate.academy.service;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger();
+
     @Override
-    public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+    public User login(String login, String password) {
+        logger.info("Login method was called with user: {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("Failed login attempt for user '{}': incorrect password.", login);
         }
```

Comment:
The `login` method should throw an `AuthenticationException` if the password is incorrect. Currently, it only logs an error but does not throw the exception, which is a requirement according to the task description.